services/logs-forwarder/.lagoon.multi.yml

```diff
@@ -246,10 +246,12 @@ objects:
       <label @LAGOON_ELASTICSEARCH>
         <match **>
           @type elasticsearch
+          @id out_elasticsearch
           #with_transporter_log true
           #@log_level debug
           host "#{ENV['ELASTICSEARCH_HOST']}"
           port "#{ENV['ELASTICSEARCH_HOST_PORT']}"
+          scheme https
```

Comment:
we need to make this configurable via an ENV variable (while the default is `http`).
If ES is running within the same OpenShift as the logs-forwarder we're connecting directly to the ES service which is running http. Only if we have an external EKS (like in CH1 now) we use https

Reply:
ok, done